guides/common/attributes.adoc

```diff
@@ -97,6 +96,11 @@ ifeval::["{build}" == "foreman"]
 :ansible-namespace: `theforeman.foreman`
 :ansible-namespace-example: `theforeman.foreman._module_name_`
 :awx: AWX
+:BaseURL: https://docs.theforeman.org/master/
+:ManagingHostsDocURL: {BaseURL}Managing_Hosts/index-foreman.html#
+:ConfiguringAnsibleDocURL: {BaseURL}Configuring_Ansible/index-foreman.html#
+:BaseURL: https://access.redhat.com/documentation/en-us/red_hat_satellite/{ProductVersion}/html/
+// Remove the above BaseURL attribute when work with URLs is ready. This attribute redefines the BaseURL attribute to ensure that links that only use {BaseURL} work, we won't need that when all links use a specific-to-guide attribute instead
```

Comment:
So this hack is there to be able to pass link validator, or what is the reason?

Reply:
You are correct. But not only to pass the validator, more like to keep the links working. Many links use the BaseURL attribute now, so I cannot simply remove it. I will work on all links one by one and in the end all links would use the relative attribute, e.g. `ConfiguringAnsibleDocURL` in them, so it then would be possible to remove the BaseURL attribute without any issue.